### DISCUSSION TITLE
Versions compatibility with VSCode

### DISCUSSION THREAD

**Discussion by Malthaldar** [microsoft/vscode-cpptools#7889] [Q&A]
Hiya !

I am unfortunately limited to use the VScode version 1.52.1 but none of the provided versions of C/C++ extension is compatible with this version anymore. I have to install the extension offline with the vsix file.

- Where can I access older versions of the extension? (oldest one on the page is 1.3.1 and it's not working for me)
- Where do I check which version of the extension is compatible with the VScode versions?

Thanks,
Malthaldar

**Comment by sean-mcmanus** [comment]
The last release that works with 1.52.1 is https://github.com/microsoft/vscode-cpptools/releases/tag/1.2.2 .
